e2e/nomostest/config-sync.go

```diff
@@ -1287,9 +1287,10 @@ func resetRootRepos(nt *NT, upstream string, sourceFormat filesystem.SourceForma
 			resetRootRepo(nt, name, nt.RootRepos[name].UpstreamRepoURL, nt.RootRepos[name].Format)
 			nt.WaitForSync(kinds.RootSyncV1Beta1(), name, configsync.ControllerNamespace,
 				nt.DefaultWaitTimeout, DefaultRootSha1Fn, RootSyncHasStatusSyncCommit, nil)
-			// Now the repo is back to the initial commit, we can delete the safety check namespace
+			// Now the repo is back to the initial commit, we can delete the safety check namespace & role
```

Comment:
The commit message states the safety-check is for cluster-scoped resources. Should we use ClustrRole instead?

Reply:
Good catch. Fixed.

Reply:
Turns out that was causing test failures too, for hierarchical mode, because it was in cluster/ not namespaces/